### PR TITLE
Update CompoundSchema.mcf

### PR DESCRIPTION
**Pull request by mengzhensun**
Added schema for ABCD antibody dataset.

### CODE REVIEW

**Comment by mengzhensun** [comment]
> Can we make this description more specific? What do you mean by antibody type?

Added examples.

**Comment by mengzhensun** [comment]
> Are these links out to existing entities for proteins, chemical compounds, and other antigen types? The way that the description is phrased makes it sound like the property value should be an enum.

Actually I only imported the antibodies of which the antigen is protein type. I have deleted the unused schema.

File: schema/CompoundSchema.mcf [hunk 1]
```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
...
+name: "abcdID"
+description: "The accession identifier of the antibody in the ABCD antibody database. "
+rangeIncludes: schema:Text
+domainIncludes:dcs:Antibody
+descriptionUrl:"https://web.expasy.org/abcd/"
+
+Node: dcid:patentID
+typeOf: schema:Property
+name: "patentID"
+description: "The identifier of patent in PATENTSCOPE. Using PATENTSCOPE you can search 90 million patent documents including 3.9 million published international patent applications (PCT)."
```
Comment (pradh) [inline schema/CompoundSchema.mcf:643]: This seems too specific a description for a general sounding property like "patentID".

Does it make sense to make the property more specific, like "patentScopeID"?

File: schema/CompoundSchema.mcf [hunk 2]
```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
...
+name: "recognizesAntigen"
+description: "The targeting antigen of the antibody."
+rangeIncludes: dcs:Protein
+domainIncludes:dcs:Antibody
+# rangeIncludes proteins with links to UniProt, chemicals with IDs to ChEBI and a few samples to other databases (InterPro, NCBI, Prosite, UniProt-Subcell).
+
+Node: dcid:antigenType
+typeOf: schema:Property
+name: "antigenType"
+description: "The type of the antigen for an antibody. There are types as protein, chemical and the others. "
```
Comment (pradh) [inline schema/CompoundSchema.mcf:606]: Not sure what the second sentence means. Is there a typo?

File: schema/CompoundSchema.mcf [hunk 3]
```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
...
+Node: dcid:Antigen
+name: "Antigen"
+typeOf: schema:Class
+subClassOf: dcs:ChemicalCompound
+description: "The invading substances from the outer source which causes the immune response. It refers to a molecule or molecule structure which may interact with the immune cell antigen receptor or antibody. "
+descriptionUrl:"https://en.wikipedia.org/wiki/Antigen"
+
+Node: dcid:recognizesAntigen
+typeOf: schema:Property
+name: "recognizesAntigen"
```
Comment (pradh) [inline schema/CompoundSchema.mcf:597]: I'm slightly wondering about the naming of this property, specifically "recognizes". Is the target the "recognizedAntigen" or "targetedAntigen"?

File: schema/CompoundSchema.mcf [hunk 4]
```diff
@@ -578,3 +578,150 @@ description: "A reliability score is manually set for all genes and indicates th
...
+description: "The targeting antigen of the antibody."
+rangeIncludes: dcs:Protein
```
Comment (pradh) [inline schema/CompoundSchema.mcf:599]: Also puzzled that the range is not Antigen. Intentional?

**Pull request merged**